internal/dump/dump.go

```diff
@@ -0,0 +1,40 @@
+package dump
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v8"
+	"go.riyazali.net/sqlite"
+)
+
+type dump struct {
+	rdb *redis.Client
+}
+
+func (f *dump) Args() int           { return -1 }
+func (f *dump) Deterministic() bool { return false }
+func (f *dump) Apply(ctx *sqlite.Context, values ...sqlite.Value) {
+	var (
+		key    string
+		result *redis.StringCmd
+		err    error
+	)
+	if len(values) == 1 {
```

Comment:
let's make this check
```suggestion
	if len(values) >= 1 {
```

additional arguments can just be ignored rather than causing an error. You'd need to also account for the `len(values) < 1` case though